README.md

```diff
@@ -46,6 +46,9 @@ from connect.models.exception import FulfillmentFail, FulfillmentInquire, Skip
 # set logger level / default level ERROR
 logger.setLevel("DEBUG")
 
+# If we remove this line, it is done implicitly
```

Comment:
For example - better to do not bloat code with some defaults (logger, config, etc) - less code -> simpler to understand

Reply:
You want me to remove those two lines then? The good thing of having them there is that they serve as indication on how to set logger level and how to explicitly initialize (maybe with a different file name).

Reply:
Or maybe remove them from the README to make it as simple as possible, but leave it on example.py?

Reply:
remove Config and debug

Reply:
Done.